pkg/kor/kor.go

```diff
@@ -199,6 +199,7 @@ func formatOutputForResource(resource string, resources map[string][]string, opt
 			index++
 		}
 	}
+	table.Append([]string{})
 	table.Render()
 	return fmt.Sprintf("Unused %ss:\n%s", resource, buf.String())
```

Comment:
This is the affect I want `return fmt.Sprintf("Unused %ss:\n%s\n", resource, buf.String())`

Reply:
Fixed